### PR TITLE
Projects cypress specs

### PR DESCRIPTION
**Pull request by apoorv1316**
## Summary
Added cypress specs for the projects page for both admin and employee roles.

**Admin Role specs**
- should have a header and add a new project button
- should display all the project list
- should be able to add a new project
- should be able to edit a project
- should be able to delete a project

**Employee role specs**

- should have a header and display all projects list
- should not have an add new project button
- should not have edit and delete icons

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking and retains same functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code

### CODE REVIEW

**Review by mohinid — COMMENTED** [review]
Hi you have written test only for admin, is it intentional or you will be adding for other roles as well?

**Comment by apoorv1316** [comment]
> Hi you have written test only for admin, is it intentional or you will be adding for other roles as well?

Ahh forgot to commit those. Will ping you when it's ready for review.

**Comment by apoorv1316** [comment]
@mohinid Please review it now.

**Review by mohinid — COMMENTED** [review]
@apoorv1316  please add description to the pr

**Review by mohinid — CHANGES_REQUESTED** [review]
Please add tests and description in the pr.

**Comment by apoorv1316** [comment]
@mohinid Updated the description and added tests to it.

File: cypress/e2e/projects/admin.spec.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  });
+
+  it("should display all the project list", function () {
+    cy.get(projectsSelectors.projectsList).should('be.visible');
+  })
+
+  it("should be able to add a new project", function (){
+    cy.get(projectsSelectors.addNewProjectButton).should('be.visible').click();
+    // Select the first client
+    cy.get('select').select(1);
```
Comment (mohinid) [inline cypress/e2e/projects/admin.spec.js:29]: why are we using select 1? cant use the exact locator?

**Review by mohinid — CHANGES_REQUESTED** [review]

**Review by mohinid — APPROVED** [review]

**Pull request merged**